src/Functions/formatReadableDecimalSize.cpp

```diff
@@ -0,0 +1,26 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/formatReadable.h>
+
+
+namespace DB
+{
+
+namespace
+{
+    struct Impl
+    {
+        static constexpr auto name = "formatReadableDecimalSize";
+
+        static void format(double value, DB::WriteBuffer & out)
+        {
+            formatReadableSizeWithDecimalSuffix(value, out);
+        }
+    };
+}
+
+REGISTER_FUNCTION(FormatReadableDecimalSize)
+{
+    factory.registerFunction<FunctionFormatReadable<Impl>>();
```

Comment:
Also, documentation for this new function should be added similarly to, for example, BLAKE3 function:
https://github.com/ClickHouse/ClickHouse/blob/4269050a9bef7cd6cb1dcc088f3cccab773793e8/src/Functions/FunctionsHashing.cpp#L46-L57

Reply:
I've documented the function following the same way as `formatReadableSize`. Should we also document in the function like in the BLAKE3 example

Reply:
Yes, otherwise FastTest will fail

Reply:
Documented function